### ISSUE TITLE
Failure to create existing directory

### ISSUE THREAD

**Issue by greole** [gerlero/apt-install#41]
We currently observe an issue with setup-foam
```
Download packages to cache
  mkdir: cannot create directory ‘.apt-install/packages’: File exists
  Error: Process completed with exit code 1.
```
See 
https://github.com/exasim-project/NeoFOAM/actions/runs/11880294674/job/33103383610?pr=81

Do you have an idea what the issue is?

**Comment by gerlero** [comment]
@greole Thanks for the report! I've transferred the issue here as I believe this package is the culprit. I'll be releasing a fix ASAP.

**Comment by gerlero** [comment]
The issue should be fixed by #42 here, which is already released. Let me know if that doesn't do it.
